FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Add RISCV_cva6 FreeRTOS Demo

Add RISCV_cva6 FreeRTOS Demo

Description
-----------
Add the Demo for RISCV_cva6. We have 2 set of test. 
1. main_blinky
2. main_full (Have used Testrunner.c from Template to enable this test)
Both test passes on Genensys FPGA HW with CVA6 core.

Test Steps
-----------
Listed in README.md

Related Issue
-----------
None
No changes outside Demo directory (FreeRTOS-Partner-Supported-Demos/). 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
